README.md

```diff
@@ -1,86 +1,173 @@
 
-## Getting Started
+Atomic App is used to bootstrap packaged container environments and run them on multiple container orchestrators. It is designed from the ground-up to be portable and provider pluggable.
 
-Atomic App itself is packaged as a container. End-users typically do not install the software from source. Instead use the `atomicapp` container as the `FROM` line in a Dockerfile and package your application on top. For example:
+  - __A "packaged installer" for all container-based environments and applications.__ Replace all those bash and Ansible scripts with one container-based deployment tool.
 
+  - __Target multiple providers:__ Specify the provider you want the Atomic App to run on. It supports Kubernetes, OpenShift, Mesos+Marathon and Docker.
+
+  - __Inherit already packaged containers:__ Create composite applications by referencing other Nulecule-compliant applications. For example, plugging in an alternative well-orchestrated database in another referenced container image.
+
+  - __Fetch and run entire environments:__ Use `atomicapp fetch` and `atomicapp run` to run pre-packaged Nuleculized containers.
+
+## Installing Atomic App
+From Linux:
+```sh
+git clone https://github.com/projectatomic/atomicapp && cd atomicapp
+make install
 ```
-FROM projectatomic/atomicapp
 
-MAINTAINER Your Name <you@example.com>
+_or_ 
 
-ADD /Nulecule /Dockerfile README.md /application-entity/
-ADD /artifacts /application-entity/artifacts
+Download a pre-signed .tar.gz from [download.projectatomic.io](https://download.projectatomic.io) / [GitHub](https://github.com/projectatomic/atomicapp/releases):
+```sh
+export RELEASE=0.4.2
+wget https://github.com/projectatomic/atomicapp/releases/download/$RELEASE/atomicapp-$RELEASE.tar.gz
+tar -xvf atomicapp-$RELEASE.tar.gz && cd atomicapp-$RELEASE.tar.gz
+make install
 ```
 
-For more information see the [Nulecule getting started guide](https://github.com/projectatomic/nulecule/blob/master/docs/getting-started.md).
+## Using Atomic App
 
-## Developers
+Prerequisite: Before proceeding, make sure you either have a Kubernetes, OpenShift or Docker environment setup and ready.
 
-First of all, clone the github repository: `git clone https://github.com/projectatomic/atomicapp`.
 
-### Installing Atomic App locally
-Simply run
+### Getting started
 
-```
-make install
-```
+Atomic App can be used either natively on your OS or ran via the [atomic cli](https://github.com/projectatomic/atomic) command. 
 
-If you want to do some changes to the code, I suggest to do:
+Detailed instructions on [getting started](docs/start_guide.md) are available. Alternatively, use the quickstart below.
 
-```
-cd atomicapp
-export PYTHONPATH=`pwd`:$PYTHONPATH
-alias atomicapp="python `pwd`/atomicapp/cli/main.py"
+
+### Quickstart
+
+__Running Apache on Docker:__
+```sh
+▶ sudo atomicapp run --provider=docker projectatomic/helloapache
```

Comment:
for these example commands let's make them compatible with the Atomic cli notation - I think with atomic cli we have to put the image name first and then the option after right?

so we should changes these examples to put the `--provider=` after the image name. Does that seem like the right thing to do?